apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -21,6 +24,39 @@ import { useAppState } from '@aragon/api-react'
 import BigNumber from 'bignumber.js'
 import { displayCurrency } from '../../utils/helpers'
 
+const PendingStatus = (theme) => (
+  <div style={{ display: 'flex' }}>
+    <IconClock style={{
```

Comment:
Can we use `css=` in all of these spots instead of `style=`? That will use `styled-components` behind the scenes, which injects `style` tags into the `head` of the document, rather than using inline styles. This results in better browser performance and is easier to work with.

Reply:
Whenever I write the `css` property in the custom `<Icon>` element, it fails with
![Screenshot from 2019-11-15 14-31-37](https://user-images.githubusercontent.com/16065447/68971410-787ba480-07b7-11ea-9462-70a0fe150d48.png)
So in that case I had to leave the `style` prop. Unless you know of a way to solve it, that is.